### ISSUE TITLE
pmd rule link not processed in rule description

### ISSUE THREAD

**Issue by stokpop** [jborgers/sonar-pmd#611]
For pmd:UnnecessaryBoxing there is an unresolved rule reference:

```
are reported by {% rule UnnecessaryCast %} instead.
```
